### PR TITLE
Update Headline & Subheading colour to Shadow

### LINKED ISSUE #279: Update appearance of psammead-headings
**Is your feature request related to a problem? Please describe.**
UX have updated the designs for the headline & subheading components.

Depends on #277.

**Describe the solution you'd like**
Change the font colours for both headline & subheadline to `#3F3F42`, or "Shadow". No other changes to appearance should be made at this time.

Change subheadings in psammead-headings to use `Reith Sans Bold`.

As this creates a visible change, I think this would constitute a "breaking" release. Seeing as we're sub-1.0, this would increment the minor field.

**Describe alternatives you've considered**

**Testing notes**
Check to make sure colours and font are as expected. Also, check the colour ratio passes. As this is in Psammead, this will not require Cypress tests.

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

**Additional context**

- [x] Initially labelled with ["Refinement needed"](https://github.com/bbc/psammead/labels/Refinement%20Needed)

### PR DESCRIPTION
**Pull request by sareh**
Resolves #279

**Overall change:** Update Headline & Subheading colour to Shadow `'#3F3F42'` & Subheading updated to use Reith Sans Bold font.

Screenshots of Storybook for Headline, with dev tools showing color is Shadow & font is unchanged:
screen shot of heading

Subheading component shows color is Shadow and font updated to Reith Sans Bold:
screen shot of subheading

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

**Review by dr3 — CHANGES_REQUESTED** [review]
Does this PR resolve "Change subheadings in psammead-headings to use `Reith Sans Bold`." from the issue https://github.com/bbc/psammead/issues/279?

**Comment by sareh** [comment]
@dr3 Yes. I've pushed that change.

**Review by dr3 — DISMISSED** [review]

File: .storybook/config.js [hunk 1]
```diff
@@ -43,6 +43,13 @@ const GlobalStyle = createGlobalStyle`
     font-weight: 600;
     src: url('https://gel.files.bbci.co.uk/r2.302/BBCReithSerif_W_Md.woff2') format('woff2'), url('https://gel.files.bbci.co.uk/r2.302/BBCReithSerif_W_Md.woff') format('woff');
   }
+  @font-face {
```
Comment (sareh) [inline .storybook/config.js:46]: I added this so the Reith Sans Bold font will load in Storybook. Though issue https://github.com/bbc/psammead/issues/285 updating font-faces within Psammead utilities will be used to update the font-faces in the Storybook config.

Comment (dr3) [reply .storybook/config.js]: Do you need to update the screenshot in this PR description?

File: .storybook/config.js [hunk 2]
```diff
@@ -43,6 +43,13 @@ const GlobalStyle = createGlobalStyle`
...
+    font-display: optional;
+    font-family: ReithSerifNewsBold;
```
Comment (dr3) [inline .storybook/config.js]: 🙈 This is serif

**Review by dr3 — APPROVED** [review]

**Review by bcmn — APPROVED** [review]
Looks good in Storybook -- slightly concerned about making sure we guarantee we've updated the bold to be correct following the proper implementation, though. Is there an issue for this, @sareh?

**Comment by sareh** [comment]
@bcmn Thanks. I've opened this issue to capture this work: https://github.com/bbc/psammead/issues/312

**Comment by jamesbrumpton** [comment]
👍

**Pull request merged**